### DISCUSSION TITLE
Clone primary sources source modules

### DISCUSSION THREAD

**Discussion by physikerwelt** [MaRDI4NFDI/docker-quickstatements#1] [Ideas]
@aot29 I think one alternative to the fetcher would be to fork the repositories to the MaRDI GitHub space. That way, we can ensure that the version that we have been using stays available.

**Comment by aot29** [comment]
Agree. But first I want to have the Docker images pushed by CI to the container registry.

**Reply by physikerwelt** [reply]
Sure. I did that only once. See https://github.com/physikerwelt/mediawiki-docker/blob/master/.github/workflows/docker-publish.yml

**Reply by aot29** [reply]
Quickstatements image:
ghcr.io/mardi4nfdi/docker-quickstatements:master
